logging/access.go

```diff
@@ -113,7 +113,7 @@ func stripQueryString(u string) string {
 }
 
 // Logs an access event in Apache combined log format (with a minor customization with the duration).
-func LogAccess(entry *AccessEntry) {
+func LogAccess(entry *AccessEntry, metadata map[string]string) {
```

Comment:
->
`func LogAccess(entry *AccessEntry, metadata logrus.Fields) {`

Maybe rename `metadata` to `additional`

Reply:
Wouldn't change here `metadata` type to `logrus.Fields`, because I don't think it's a good idea expose the internals of the internal logger library used to places like proxy.  Changing name to `additional` makes a lot of sense

Reply:
Then please do map[string]interface{}